lib/mix/lib/mix/cli.ex

```diff
@@ -126,4 +133,13 @@ defmodule Mix.CLI do
       ["--version", "-v"], do: :version
 
   defp check_for_shortcuts(_), do: nil
+
+  defp usage() do
```

Comment:
ok. I can work on a PR.

it will look different that the other  commands, I mean the yellow background in the title and the colors, would that be ok?